pkg/meta/dynamodb/dynamodb_internal_test.go

```diff
@@ -14,13 +15,18 @@ import (
 
 	"zotregistry.io/zot/pkg/log" //nolint:go-staticcheck
 	"zotregistry.io/zot/pkg/meta/version"
+	tskip "zotregistry.io/zot/pkg/test/skip"
 )
 
 func TestWrapperErrors(t *testing.T) {
-	const (
-		endpoint = "http://localhost:4566"
-		region   = "us-east-2"
-	)
+	tskip.SkipS3(t)
```

Comment:
As far as I can tell the check and endpoint should be for DynamoDB

Reply:
yes